roles/tftp-server/defaults/main.yml

```diff
@@ -9,6 +9,14 @@ apt_proxy: false
 # Defaults to deb.debian.org | archive.ubuntu.com
 #mirror: deb.debian.org
 
+web_root: /var/www/html
+
 user_name: videoteam
-user_password: null
-pw_salt: U1ESiTXUQE0tFh2r4LRl
```

Comment:
This change needs to be made in more places (inventory/group_vars/all, roles/users/defaults/main.yml)